ansible-scylla-node/templates/scylla.yaml.j2

```diff
@@ -53,11 +53,25 @@ client_encryption_options:
     keyfile: {{ scylla_ssl.cert_path }}/{{ inventory_hostname }}.pem
     truststore: {{ scylla_ssl.cert_path }}/{{ scylla_cluster_name }}-ca.crt
 {% endif %}
-
```

Comment:
Empty line hunk - please, remove.
This is not a "cleanup PR".

Reply:
Done.